docs/howtos/DataPlane Generation - DPG/03convenient.mdx

```diff
@@ -250,7 +300,37 @@ using Azure.ClientGenerator.Core;
 <TabItem value="csharp" label="CSharp" >
 
 ```csharp
-FIXME
+// If a model is input-only, there will be one public constructor with all required properties as parameter.
+// And the required properties without setter. Optional properties will have both getter and setter
```

Comment:
```suggestion
// The required properties only have getter. Optional properties have both getter and setter.
```

Reply:
Should we also mention about the setter of collections?

Reply:
add part for collection properties